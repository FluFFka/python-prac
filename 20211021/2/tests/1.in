:asd x str(x)*((sin:=1)-round(cos(x))+x**5)
:qwe 'qwe_function'
asd 2
qwe
quit "{}:{}"
