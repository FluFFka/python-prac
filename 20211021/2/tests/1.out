222222222222222222222222222222222
qwe_function
3:5
